test/gtest/ucs/test_math.cc

```diff
@@ -203,6 +203,34 @@ UCS_TEST_F(test_math, for_each_bit) {
     EXPECT_EQ(UCS_BIT(63), gen_mask);
 }
 
+UCS_TEST_F(test_math, for_each_submask) {
+    /* Generate mask values to test */
+    std::vector<int64_t> masks;
+    masks.push_back(0);
+    masks.push_back(1);
+    masks.push_back(65536);
+    for (int i = 0; i < 100; ++i) {
+        masks.push_back(ucs::rand() % 65536);
+    }
+
+    for (std::vector<int64_t>::iterator iter = masks.begin();
```

Comment:
`std::vector<int64_t>::const_iterator`, sicne the code doens't change a content referenced by `iter`